### ISSUE TITLE
Popout sizes aren't properly being restored

### ISSUE THREAD

**Issue by RomainLK** [RomainLK/ytc-filter#52]
Somehow, when pressing "show/hide settings" several time, the sizes differs

**Comment by EvilLivesHere** [comment]
**Info**: Chrome, extension v2.0.2 
**Observation**:
- When I click "Show Settings" it sets the window to whatever fullPopout has, but then sets the fullPopout to a lower number.
- When I click "Hide Settings" it sets the window to whatever compactPopout has, but then sets compactPopout to a lower number.


For example, here's the output of my fullpopout, compactPopout, and window values after each button press.

Starting with the settings area closed, I run this script after each button press.
```
chrome.storage.local.get("@@vwe-persistence", a=> {
     console.log("window.outerHeight: " + window.outerHeight);
     console.log("window.outerWidth: " + window.outerWidth);
     console.log("FullPopout: " + JSON.stringify(a["@@vwe-persistence"].global.fullPopout));
     console.log("CompactPopout: " + JSON.stringify(a["@@vwe-persistence"].global.compactPopout));
   });
```

**START**
window.outerHeight: 465
window.outerWidth: 1356
FullPopout: {"height":476,"width":1277}
CompactPopout: {"height":426,"width":1340}

**After pressing "Show Settings"**
window.outerHeight: 476
window.outerWidth: 1277
FullPopout: {"height":437,"width":1261}
CompactPopout: {"height":426,"width":1340}

**After pressing "Hide Settings"**
window.outerHeight: 426
window.outerWidth: 1340
FullPopout: {"height":437,"width":1261}
CompactPopout: {"height":387,"width":1324}


**After pressing "Show Settings" again**
window.outerHeight: 437
window.outerWidth: 1261
FullPopout: {"height":398,"width":1245}
CompactPopout: {"height":387,"width":1324}

**Comment by RomainLK** [comment]
Fixed in 2.1.0
